### ISSUE TITLE
Project name and project directory name discrepancy

### ISSUE THREAD

**Issue by soininen** [Spine-project/Spine-Toolbox#1818]
It can be quite confusing to have a different name for the project than its directory name. It can also be difficult to find the project. Maybe the project name should be its directory name.

**Comment by PekkaSavolainen** [comment]
> and also show the path in the file recent after directory name (I have multiple same names in my current recent list and I don't which ones they are).

The path is shown as tooltip in the File -> Open recent list. Whenever I make copies of projects and see the same project names in the File -> Open recent list, I just open the projects, and File -> Rename project... Takes a couple of seconds.

**Comment by PekkaSavolainen** [comment]
But we should change the project name to the new directory name when one does File -> Save Project As... I think that would help a lot here. Now it keeps the old project name which can be confusing. Is that what you had in mind for this issue @soininen?

**Comment by soininen** [comment]
> But we should change the project name to the new directory name when one does File -> Save Project As... I think that would help a lot here. Now it keeps the old project name which can be confusing.

Then why not just use the directory name as the project name and drop a separate project name altogether? I think it is redundant.

**Comment by PekkaSavolainen** [comment]
Yes, I think you're right @soininen, a separate project name isn't necessary.

**Comment by PekkaSavolainen** [comment]
Done. Please click the brand new `Clear` button in `File -> Open recent` menu if you find any odd entries in the list.
